package/MDAnalysis/lib/distances.py

```diff
@@ -1334,15 +1551,33 @@ def calc_distance(a, b, box=None):
 
 
 def calc_angle(a, b, c, box=None):
-    """Angle (in degrees) between a, b and c, where b is apex of angle
+    """Calculates the angle (in degrees) between the positions `a`, `b`, and
+    `c`, where `b` represents the apex of the angle::
+
+            b---c
+           /
+          a
+
+    If the optional argument `box` is supplied, periodic boundaries are taken
+    into account when constructing the connecting vectors between coordinates,
+    i.e., the minimum image convention is applied for the vectors forming the
+    angle. Either orthogonal or triclinic boxes are supported.
 
     Parameters
     ----------
-    a, b, c : numpy.ndarray
-        single coordinate vectors
-    box : numpy.ndarray
-        simulation box if given periodic boundary conditions will be applied to
-        the vectors between atoms
+    a,b,c : numpy.ndarray
+        Single position vectors of shape ``(3,)``.
+    box : numpy.ndarray, optional
+        The unitcell dimensions of the system, which can be orthogonal or
+        triclinic and must be provided in the same format as returned by
+        :attr:`MDAnalysis.coordinates.base.Timestep.dimensions`:\n
+        ``[lx, ly, lz, alpha, beta, gamma]``.
+
+    Returns
+    -------
+    angle : float
+        The angle between the vectors `b`:math:`\\rightarrow`\ `a` and
+        `b`:math:`\\rightarrow`\ `c`.
```

Comment:
Maybe add a 
```reST
See Also
--------
calc_angles : calculate multiple angles efficiently
```

Reply:
That is something I deliberately omitted. Since I introduced the new `@check_coords()` decorator, both `calc_angle()` and `calc_angles()` accept single coordinates as well as coordinate arrays, and return either an array or a single float, depending on the input.
The only difference is that `calc_angle()` returns the angle(s) in degrees while the result of `calc_angles()` is in radians. IMHO we should deprecate `calc_angle()`.
See also my comments on `calc_distance()` and `calc_dihedral()`.